machine/alphanetworks/alphanetworks_snc60a0_486f/rootconf/sysroot-lib-onie/sysinfo-platform

```diff
@@ -0,0 +1,25 @@
+
+get_serial_num()
```

Comment:
This override of get_serial_num() is not necessary.

The definition here is the same as the CPU architectural default found in rootconf/x86_64/sysroot-lib-onie/sysinfo-arch.